Tests/UnitTests/FileShareService.DesktopClientTests/Modules/Admin/NewBatchJobViewModelTests.cs

```diff
@@ -474,10 +477,13 @@ public void TestFileSearchWithMacroInDirectory(string directoryMacro, string exp
                 () => fakeFileShareApiAdminClient,
                 fakeCurrentDateTimeProvider, macroTransformer, dateTimeValidator, fakeMessageBoxService);
 
-
             Assert.AreEqual(2, vm.Files.SelectMany(f => f.Files).Count());
-            CollectionAssert.AreEqual(new[] { file1FullFileName, file2FullFileName },
+            CollectionAssert.AreEqual(new[] { filesA.expandedFilePath, filesB.expandedFilePath },
                 vm.Files.SelectMany(f => f.Files.Select(fi => fi.FullName)));
+
+            vm.ExcecuteJobCommand.CanExecute();
+
+            Assert.IsEmpty(vm.ValidationErrors);
```

Comment:
I think this should do the same thing:
```suggestion
            Assert.IsTrue(vm.ExcecuteJobCommand.CanExecute());
```

Reply:
I agree. Thanks